py_mmd_tools/mmd_util.py

```diff
@@ -0,0 +1,108 @@
+"""
+Utility tool to work on mmd xml files.
+
+ License:
+     This file is part of the S-ENDA-Prototype repository (https://github.com/metno/py-mmd-tools).
+     S-ENDA-Prototype is licensed under GPL-3.0 (https://github.com/metno/py-mmd-tools/blob/master/LICENSE)
+"""
+
+import lxml.etree as ET
+import pathlib
+import time
+
+import confuse
+from confuse.exceptions import NotFoundError    
+
+import logging
+
+def get_logpath():
+    try:
+        config = confuse.Configuration('mmdtool', __name__)
+        logfilepath=config['paths']['logs'].get()
+    except NotFoundError:
+        logfilepath = './logs/'
+    if not pathlib.Path(logfilepath).exists():
+       pathlib.Path(logfilepath).mkdir(parents=True, exist_ok=True)
+    return logfilepath
+
+
+
+import logging
```

Comment:
already imported - it's a good practice to have all imports at the top of the file to avoid repetition

Reply:
All the imports are at the top, that was obviously left there and repeated by-mistake after adding the `get_logpath` method - Did you find any other misplaced import?

Reply:
No, I didn't see any..